app/models/grda_warehouse/report/base.rb

```diff
@@ -153,8 +154,8 @@ def self.sh_columns
         she_t[:project_type].to_sql,
         she_t[:project_tracking_method].to_sql,
         o_t[:id].as('organization_id').to_sql,
-        she_t[:housing_status_at_entry].to_sql,
-        she_t[:housing_status_at_exit].to_sql,
+        e_t[:LivingSituation].to_sql,
+        ex_t[:HousingAssessment].to_sql,
```

Comment:
Just confirming that this should not be`Destination`. Seems odd that at entry would be 3.917, but at exit is W5?

Reply:
That is my reading of the notes we had.  (And that's what we were doing.)